reactioncommerce/reaction
# refactor: use npm package for simple-pricing plugin

Resolves https://github.com/reactioncommerce/reaction/issues/6134

This PR removes the internal `simple-pricing` plugin, and installs the new `api-plugin-pricing-simple` from npm as a replacement.

The code in that npm plugin is the same, so the expected outcome of this PR is seeing nothing different than how the app currently runs.

## Testing
- Start reaction, see that `simple-pricing` is the correct version in the `systemInformation` query
- Update a price for a product, publish, and see the changes in the UI

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
